backend/app/routers/order.py

```
@@ -229,45 +242,45 @@
         "status": order.status,
         "created_at": order.created_at,
         "bill_amount": bill.amount if bill else None,
-        "order_items": order_items
+        "order_items": order_items,
     }
     return result
```

Ruff (0.8.2)

Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)